### DISCUSSION TITLE
Set Bonuses and Passives Interface

### DISCUSSION THREAD

**Discussion by Malphaze** [runelite/runelite#16238] [Ideas]
**_THIS IS A COPY AND PASTE I DID ON OSRS FORUMS, I AM BRANCHING IT OUT AND TOO LAZY TO REWRITE BUT EXPLAINS THE TOPIC PERFECTLY FINE_** 

Hello! 

I just realized and noticed there are _Set Bonuses_ and _Passives_ in OldSchool Runescape. 

Why is there no Runelite Plugin for this and or why hasn't the Oldschool Runescape Devs implemented a interface for weapon inspecting or interface for items in the game if they have a possible Set Bonus or Passive effect? I feel like that this should be a thing in the game and makes things easier having to google constantly for things such as this and I'm sure many would agree.

Who do I contact for this suggestion when it comes to OSRS Devs and Runelite Creators?

World of Warcraft has a interface implement that is really easy and neat to see and read over. Here are some examples that hopefully in the future if and when it's made, it can take some ideas off the pictures? 

- **The Set Bonuses**
https://gyazo.com/0ac050c477a1cdc7c6af577ea041b500

- **The Passive**
https://gyazo.com/758ba72d86d8e1b00ca3c1851e33dcac

**Comment by AFKLgnd** [comment]
There is a way to see set bonuses in OSRS. In the worn equipment tab if you click view equipment stats there is a set bonuses button. Unlike WoW there is a lot more emphasis on switching gear in RS so adding these to a tooltips would probably increase screen clutter.

See image below:
https://i.imgur.com/frgWZgy.png

**Reply by Malphaze** [reply]
Didn't know about that, but thing is, what if someone doesn't own this piece or is deciding to buy it? I would still like to prefer to see the bonus before buying something if it has a passive or bonus effect. 

Screen clutter wouldn't be a issue considering it's made decently and it's only to those who choose to use the plugin, not everyone would have to use it. 

But I understand your POV though.
